src/utils/db.js

```diff
@@ -0,0 +1,10 @@
+import { Sequelize } from 'sequelize';
+import 'dotenv/config';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  database: process.env.DB_DATABASE,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  dialect: 'postgres',
+});
```

Comment:
The Sequelize constructor is missing the 'port' option. If your database is running on a non-default port, you should include 'port: process.env.DB_PORT' in the configuration.